lib/tlsca/ca.go

```diff
@@ -399,6 +406,14 @@ var (
 	// allowed Azure identity into a certificate.
 	AzureIdentityASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 1, 17}
 
+	// AppGCPServiceAccountASN1ExtensionOID is an extension ID used when encoding/decoding
+	// GCP service account into a certificate.
+	AppGCPServiceAccountASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 1, 18}
+
+	// GCPServiceAccountASN1ExtensionOID is an extension ID used when encoding/decoding
+	// allowed GCP service account into a certificate.
+	GCPServiceAccountASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 1, 19}
```

Comment:
I was following existing docs: https://github.com/gravitational/teleport/blob/6fa75978e4f7986a16e2ee57725b4323f2f83af5/lib/tlsca/ca.go#L374-L380

But I concur this could be improved.

Reply:
Comments improved.